backend/corpora/api_server/logger.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+from logging import LogRecord
+from logging.config import dictConfig
+from typing import List
+
+from pythonjsonlogger import jsonlogger
+from pythonjsonlogger.jsonlogger import merge_record_extra
+
+
+# The fields to log using the json logger.
+LOGGED_FIELDS = ["levelname", "asctime", "name", "message"]
+LOG_FORMAT = " ".join([f"%({field})" for field in LOGGED_FIELDS])
```

Comment:
Is this equivalent to `" ".join(LOGGED_FIELDS)`?

Reply:
the python logger expects a format of "(field1) (field2)". This is wrapping them in `( )`